content/docs/04-clusters/06-cluster-management/14-noc-ui.md

```diff
@@ -43,10 +44,78 @@ To monitor the cluster location follow the below steps:
 
 <br />
 
-* Login to the Palette console and select `Clusters` from the left main menu.
+* Log in to the Palette console and select **Clusters** from the left main menu.
 
 
-* Go to `Map View` from the top menu.
+* Go to **Map View Icon** below the `Clusters` tab.
 
 
 The map will display all the cluster locations under that user’s scope. 
+
+# Map Filters
+
+Palette Map Filters filter out specific cluster(s) using built-in or custom filters for an enhanced user experience. This map filter provides an enhanced user experience when you have many clusters dispersed geographically across multiple scopes and clouds. You have two types of filters:- **Built in Filters** and **Custom Filters**.
+
+## Built-in filters
+
+The filters are available in the Palette console by default and can be selected from the **Add Filter** drop-down menu. You can have the following built-in filters from the Palette console.
```

Comment:
```suggestion
Built-in filters are available in the Palette console by default and can be selected from the **Add Filter** drop-down menu. You can use the following built-in filters.
```
This re-writes the sentence by specifying in the beginning what is available. The second sentence is re-written to active voice.